### PR TITLE
boilerplate

### PR DESCRIPTION
**Pull request by pandaren89**
Generator express boilerplate. 
test server already up on AWS.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,19 +1,16 @@
 {
-  "name": "integrify-system-backend",
-  "version": "0.0.1",
-  "description": "",
-  "main": "server.js",
+  "name": "myapp",
+  "version": "0.0.0",
```
Comment (ximbong91023) [inline package.json]: Should the version be 0.0.1 instead?

**Comment by pandaren89** [comment]
maybe app name is the same as the git game

File: package.json [hunk 2]
```diff
@@ -1,19 +1,16 @@
 {
-  "name": "integrify-system-backend",
-  "version": "0.0.1",
-  "description": "",
-  "main": "server.js",
+  "name": "myapp",
```
Comment (ximbong91023) [inline package.json]: We should rename the application

File: package.json (hunk 1 above)
Comment (pandaren89) [reply package.json]: version can be whatever at the moment, but when we have a version ready to test, we change it to 0.1

File: package.json (hunk 2 above)
Comment (pandaren89) [reply package.json]: maybe app name is the same as the git game

Comment (ximbong91023) [reply package.json]: I agree on that

**Comment by pandaren89** [comment]
Ive pushed the changed to the test branch
it should be able to merge now

**Pull request merged**